packages/web-components/@sfx/search-box/test/interaction/search-box.test.ts

```diff
@@ -0,0 +1,120 @@
+import { SEARCHBOX_EVENT } from '../../src/events';
+import { expect, stub, waitForUpdateComplete } from '../utils';
+import SearchBox from '../../src/search-box';
+import { TemplateResult, html } from 'lit-element';
```

Comment:
We've decided that non-local imports should be specified first when possible. For these imports, I would order it as 4, 2, 3, 1 (non-local, test utils, class under test, other supporting modules).